### PR TITLE
[PHP]: HLEB2 framework added

### PR DESCRIPTION
**Pull request by phphleb**
This PR includes the integration of the `hleb2`, `hleb2-roadrunner`, and `hleb2-swoole` variations into a single framework, HLEB2.  

Website: https://hleb2framework.ru/
Repository: https://github.com/phphleb/hleb/

### CODE REVIEW

**Comment by waghanza** [comment]
Could you make this PR as draft while working on it @phphleb ?

**Comment by phphleb** [comment]
> Could you make this PR as draft while working on it @phphleb ?

All work has already been completed. This is the final result.

File: php/hleb2-swoole/composer.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "require": {
+    "phphleb/framework": ">=v2.0.48"
```
Comment (waghanza) [inline php/hleb2-swoole/composer.json:3]:
```suggestion
    "phphleb/framework": "~2.0.0"
```

File: php/hleb2-swoole/composer.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  },
+  "autoload": {
+    "classmap": [
+      "app/"
+    ]
+  },
+  "minimum-stability": "dev"
```
Comment (waghanza) [inline php/hleb2-swoole/composer.json:10]:
```suggestion
```

File: php/hleb2-swoole/config.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+framework:
+  website: hleb2framework.ru
+  github: phphleb/hleb
+  version: 2.0.48
```
Comment (waghanza) [inline php/hleb2-swoole/config.yaml:4]:
```suggestion
  version: 2.0
```

File: php/hleb2-swoole/public/index.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+use Swoole\Http\Request;
+use Swoole\Http\Response;
+use Swoole\Http\Server;
+
+include __DIR__ . "/../vendor/autoload.php";
+
+$http = new Server('0.0.0.0', 3000, SWOOLE_BASE, SWOOLE_SOCK_TCP);
+$http->set([
+    'worker_num' => swoole_cpu_num() * 2,
```
Comment (waghanza) [inline php/hleb2-swoole/public/index.php:11]: is there a way to use a swoole config file for those options ? in order to make sure we use the same file for all swoole based

File: php/hleb2/composer.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "require": {
+    "php": ">=8.2.0",
+    "phphleb/framework": "v2.0.48"
```
Comment (waghanza) [inline php/hleb2/composer.json:4]:
```suggestion
    "phphleb/framework": "~2.0.0"
```

File: php/hleb2/composer.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  },
+  "autoload": {
+    "classmap": [
+      "app/"
+    ]
+  },
+  "minimum-stability": "dev"
```
Comment (waghanza) [inline php/hleb2/composer.json:11]:
```suggestion
```

File: php/hleb2/config.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+framework:
+  website: hleb2framework.ru
+  github: phphleb/hleb
+  version: 2.0.48
```
Comment (waghanza) [inline php/hleb2/config.yaml:4]:
```suggestion
  version: 2.0
```

**Review by waghanza — CHANGES_REQUESTED** [review]
does road-runner use the same config for each php framework with this engine ?

File: php/hleb2-swoole/public/index.php (hunk 4 above)
Comment (phphleb) [reply php/hleb2-swoole/public/index.php:11]: This is the most common type of Swoole configuration I've seen so far. It's unlikely that you'll be able to preconfigure Swoole at the Docker installation stage, as this configuration is tied to the application's runtime behavior.

**Comment by phphleb** [comment]
> использует ли road-runner одну и ту же конфигурацию для каждого PHP-фреймворка с этим движком?

Yes, from what I've found here, this configuration is standard. For example, in `php/spiral`, the configuration is exactly the same.

File: php/hleb2-swoole/public/index.php (hunk 4 above)
Comment (waghanza) [reply php/hleb2-swoole/public/index.php:11]: Yes, sure. I just meant if swoole could read an external file to be configured, an ini, XML or else

**Review by sy-records — APPROVED** [review]

File: php/hleb2-swoole/public/index.php (hunk 4 above)
Comment (phphleb) [reply php/hleb2-swoole/public/index.php:11]: > Yes, sure. I just meant if swoole could read an external file to be configured, an ini, XML or else

I completely support the idea of making configurations unified and editable in a single place, and I’m genuinely impressed with the concept and implementation of the benchmarks themselves. However, I don’t currently have any ideas on how to make the Swoole configuration fully universal. Frameworks like Laravel, for example, might still require additional adjustments due to their specific requirements. Even if we imagine manually generating identical YAML files for Swoole settings and converting each into global PHP variables to set in the configuration, differences would likely persist regardless.

**Pull request merged**